app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shlex
+
+
+def copy_file(command: str) -> None:
+    commands = shlex.split(command)
+    source_file = commands[1]
+    destination_file = commands[2]
```

Comment:
commands, source_file,  destination_file = command.split() will be better to use this way